tests/integration/services/data/project-04/extensions/eda/rulebooks/demo_vault_str.yml

```diff
@@ -0,0 +1,18 @@
+---
+- name: vaulted
+  hosts: all
+  sources:
+    - name: vaulted 
+      ansible.eda.range:
+        limit: !vault |
+          $ANSIBLE_VAULT;1.1;AES256
+          36376437373138336362373430393138396139376631643039356330386235633536336666653433
+          3262333434663238663235353933646437633430616466310a323739313739356338663038356438
+          35623138663333633638323666633861373361663831613262383630343738373033323338326230
+          3935386638343137370a626161373036383764646633656163613734343362623434366165393063
+          3736
```

Comment:
@hsong-rh Just for testing purposes we might have to use this same rulebook in the e2e test and we won't remember what vault password that was used to encrypt this string. Do you want to add a comment to indicate what secret was used to create this vaulted string. I am guessing on the server side for the E2E test we would have to create 1 test with a bad vault credential and one with a good vault credential to test the behavior